kolena/_experimental/quality_standard.py

```diff
@@ -87,17 +295,38 @@ def download_quality_standard_result(
     :param metric_groups: The names of the metric groups to include in the result.
     :param intersect_results: If True, only include datapoint that are common to all models in the metrics calculation.
     Note all metric groups are included when this value is `None`.
+    :param confidence_level: The confidence score used to calculate the Margin of Error, representing the probability of
+     capturing the true population parameter within the calculated MOE, we recommend setting between 0.9 - 0.99.
+     If this is specified performance delta column will be shown, metric valued will be classified as
+      improved, regressed, similar or unknown according to the MOE.
+    :param reference_model: The name of the model to use as a reference for the Margin of Error calculation,
+    :param reference_eval_config: The evaluation configuration to use in conjunction with the reference model,
+     if unspecified the first model in the models list will be used
+     as a reference for the Margin of Error, for None eval config, pass in 'null' instead of None. If this is None,
+     the first eval config will be used.
```

Comment:
```suggestion
    :param reference_eval_config: The evaluation configuration to use in conjunction with the `reference_model`.
        If this is `None`, the first eval config will be used. If you wish to use an evaluation configuration of `None`, please instead specify the configuration as `'null'`.
```

Also, seems a bit odd to make the user specify `null` instead of using `None` -- would it make sense to instead define it as:
```
reference_eval_config: Optional[Union[Dict[str, Any], Literal["first"]]] = "first",
```

Reply:
Yep make sense, I was just trying to distinguish between null and None